lib/tasks/claims.rake

```diff
@@ -8,11 +8,14 @@ namespace :claims do
   desc "Create demo claim data for all states, allocating to case work as required"
   task :demo_data => :environment do
 
+    ADVOCATE_COUNT = 4
+    CLAIMS_PER_ADVOCATE_PER_STATE = 3
+
     example_advocate = Advocate.find(1)
```

Comment:
should we check for failure ? if no user find fail gracefully or do something useful like create one ?